src/app/(after-login)/planner/[id]/_components/planner-synopsis-form/PlannerSynopsisForm.tsx

```diff
@@ -31,22 +33,30 @@ export default function PlannerSynopsisForm() {
         isMulti={true}
         isRequired={true}
       />
-      <Dropdown
-        name="synopsis.length"
-        type="outlined"
-        placeholder="분량"
-        label="분량"
-        options={PLANNER_SYNOPSIS_LENGTH}
-        isRequired={false}
-      />
-      <TextField name="synopsis.purpose" label="기획 의도" variant="expand" />
+      <PlannerFieldWithButton name="synopsis.length" isDropdown={true}>
+        <Dropdown
+          name="synopsis.length"
+          type="outlined"
+          placeholder="분량"
+          label="분량"
+          options={PLANNER_SYNOPSIS_LENGTH}
+          isRequired={false}
+        />
+      </PlannerFieldWithButton>
+
+      <PlannerFieldWithButton name="synopsis.purpose">
+        <TextField name="synopsis.purpose" label="기획 의도" variant="expand" />
+      </PlannerFieldWithButton>
+
       <TextField
         name="synopsis.logline"
         label="로그 라인"
         variant="expand"
         options={{ required: { value: true, message: 'required' } }}
```

Comment:
재형님, 테스트 해보니 필수 입력 항목인 **`로그 라인`** 필드를 공백으로 채워도 form이 valid가 돼서 저장할 수 있는 상태가 되는 걸 확인했습니다.

options prop에 `required`도 좋지만 추가로 **`validate`** 를 작성해 주시면 더 좋을 것 같습니다. 😌

```suggestion
 options={{
          required: { value: true, message: 'required' },
          validate: (value) => {
            if (value.trim() === '') {
              return false;
            }
          },
        }}
```

Reply:
수정했습니다!